assets/js/interactivity/store.js

```diff
@@ -35,6 +35,13 @@ const getSerializedState = () => {
 const rawState = getSerializedState();
 export const rawStore = { state: deepSignal( rawState ) };
 
+export const updateStore = ( newStore ) => {
+	rawStore.state.woocommerce = {
+		...rawStore.state.woocommerce,
+		...newStore.state.woocommerce,
+	};
+};
+
```

Comment:
This is not really needed.

Reply:
Fixed with 5fae283eaada268eeddd06f5f5189ae2745776e7.